protostar/cheatable_starknet/callable_hint_locals/warp_hint_local.py

```diff
@@ -16,10 +14,10 @@ def _build(self):
 
     def warp(
         self,
-        target_contract_address: RawAddress,
+        target_contract_address: int,
         blk_timestamp: int,
-    ) -> Callable[[], None]:
-        return self.controllers.block_info.cheat(
+    ):
```

Comment:
```suggestion
    ) -> None:
```
nit, maybe it's not the most important thing in the universe but removing the return type just feels like a step back.